Source/WebCore/platform/graphics/WidthIterator.cpp

```diff
@@ -62,18 +62,16 @@ WidthIterator::WidthIterator(const FontCascade& font, const TextRun& run, HashSe
 }
 
 struct OriginalAdvancesForCharacterTreatedAsSpace {
-    explicit OriginalAdvancesForCharacterTreatedAsSpace(GlyphBufferStringOffset stringOffset, bool isSpace, float advanceBefore, float advanceAt)
+    explicit OriginalAdvancesForCharacterTreatedAsSpace(GlyphBufferStringOffset stringOffset, bool isSpace, float advanceAt)
```

Comment:
Maybe rename "advanceAt" (and "advanceAtCharacter") to just "advance"? Since we no longer have two different advances.

Also the class name could drop the "s" in "OriginalAdvances".